PyInstaller/hooks/hook-gevent.py

```diff
@@ -0,0 +1,4 @@
+from PyInstaller.utils.hooks import collect_submodules
+
+# required for gevent.monkey.patch_all in gevent 1.1
```

Comment:
Please add a comment stating what gevent is, a link to it's homepage and which version you tested.